components/common/Header/config.js

```diff
@@ -40,54 +40,25 @@ export const icons = [
 
 const backgrounds = [
   {
-    backgroundColor: '#c099ff',
-    backgroundImage: `radial-gradient(at 2% 10%, hsla(285,85%,79%,1) 0px, transparent 50%),
-  radial-gradient(at 83% 15%, hsla(332,95%,65%,1) 0px, transparent 50%),
-  radial-gradient(at 63% 5%, hsla(239,80%,68%,1) 0px, transparent 50%),
-  radial-gradient(at 0% 61%, hsla(328,80%,77%,1) 0px, transparent 50%),
-  radial-gradient(at 39% 31%, hsla(356,80%,70%,1) 0px, transparent 50%),
-  radial-gradient(at 92% 33%, hsla(344,84%,70%,1) 0px, transparent 50%),
-  radial-gradient(at 9% 14%, hsla(197,61%,68%,1) 0px, transparent 50%);`
+  backgroundColor: 'rgb(120,2,6)',
+  backgroundImage: `linear-gradient(99deg, rgba(120,2,6,1) 21%, rgba(64,5,33,1) 49%, rgba(21,27,69,1) 89%);`
```

Comment:
why this change??

Reply:
Before background color was not matching with the navbar so i added bright colors…

Reply:
@Heena31... please could you try to reuse our color scheme..I think it is good to have the previous one. 
adding is ok...but for removing some parts there should be some correct logic.I do not found it!

I'll check on my local so I can provide some detailed comments..till the time you can try to reuse the previous scheme